docs/02-app/01-building-your-application/01-routing/08-route-groups.mdx

```diff
@@ -54,6 +55,20 @@ To opt specific routes into a layout, create a new route group (e.g. `(shop)`) a
   height="930"
 />
 
+### Opting loading skeletons to a specific route
+
+To opt a [loading skeleton](/docs/app/building-your-application/routing/loading-ui-and-streaming) via a `loading.js` file to a specific route, create a new route group (e.g., `/(overview)`) and then move your `loading.tsx` inside that route group.
```

Comment:
```suggestion
To create a [loading skeleton](/docs/app/building-your-application/routing/loading-ui-and-streaming) via a `loading.js` file for a specific route, create a new route group (e.g., `/(overview)`) and then move your `loading.tsx` inside that route group.
```

Reply:
@ijjk I don't "create" makes more sense here because the idea is you already had an existing `loading.tsx` (need to fix the file extension), and now you just want to opt it in a specific route. If we went the "create" route, we would have to re-word the "move" because it wouldn't make sense to create a `loading.tsx` and then move it—you'd just create it in the right place.

Reply:
@samcx did you see my comment? I really feel we need to better explain how loading works when you combine it with route groups and what possibilities it unlocks. This examlpe does not say that.